### PR TITLE
Remove comment

### PR DESCRIPTION
**Pull request by In-line**

### CODE REVIEW

File: rehlds/engine/world.cpp [hunk 1]
```diff
@@ -181,7 +181,7 @@ struct hull_s *SV_HullForBsp(edict_t *ent, const vec_t *mins, const vec_t *maxs,
 			else
 				hull = &model->hulls[3];
 		}
-// TODO: add cvar for this?
+// DONE: add cvar for this
```
Comment (WPMGPRoSToTeMa) [inline rehlds/engine/world.cpp]: I have forgotten what I meant by this.

File: rehlds/engine/world.cpp [hunk 2]
```diff
@@ -181,7 +181,7 @@ struct hull_s *SV_HullForBsp(edict_t *ent, const vec_t *mins, const vec_t *maxs,
 			else
 				hull = &model->hulls[3];
 		}
-// TODO: add cvar for this?
```
Comment (WPMGPRoSToTeMa) [inline rehlds/engine/world.cpp]: I have forgotten what I meant by this.

Comment (In-line) [inline rehlds/engine/world.cpp]: @WPMGPRoSToTeMa I think you meant `sv_rehlds_hull_centering` which was added later

Comment (In-line) [inline rehlds/engine/world.cpp]: @WPMGPRoSToTeMa Hm. It was added by this commit fe9bfbb. Anyway it's confusing

Comment (WPMGPRoSToTeMa) [inline rehlds/engine/world.cpp]: I think we can just remove this line.

Comment (In-line) [inline rehlds/engine/world.cpp]: @WPMGPRoSToTeMa I removed it.

**Pull request merged**